keyboards/ergodox_ez/keymaps/mac_win_german_programmer/config.h

```diff
@@ -0,0 +1,10 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+#endif
+
+#define NORMAL_MODE 0
+#define INSERT_MODE 1
+#define SYMB 2
+#define MOUSE 3
+
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.